nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-2-6-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/KafkaProcessorUtils.java

```diff
@@ -199,6 +203,13 @@
         .identifiesControllerService(KerberosCredentialsService.class)
         .required(false)
         .build();
+    public static final PropertyDescriptor KERBEROS_USER_SERVICE = new PropertyDescriptor.Builder()
```

Comment:
With this being a public variable, is it worth renaming to `SELF_CONTAINED_KERBEROS_USER_SERVICE` to indicate the more specific type of `KerberosUserService` it identifies?